theme/layouts/_header.ejs

```diff
@@ -0,0 +1,17 @@
+<header>
+    <div id="header-top" class="col2 device">
+      <a href="/" id="header-logo" class="cell1 start">
+        {attribute1 <img src="<%= site.root %>/images/logo.png" alt="Privacy Manager logo" />}
```

Comment:
Suggestion: You probably want to define a string for alternative attribute in `header.json` and reuse it here, similar to what you did for menu items in `common.json`(which might be missleading location for those items already, maybe you can move to header.json?).
 

Note: The [syntax you are using here](https://cmints.io/en/documentation/i18n/#translation-strings), is usually used for pages content, where you can define stringId and description right in the page. If you use that syntax for website blocks that are shared by whole website, like header, footer, sidebar and etc., then you might end up adding translation to each separate page you have, which is not what you probable want to do here.

Example:
```js
// header.json
{
  "pm-logo-alt": {
    "description": "Image alt text",
    "message": "Privacy Manager logo"
  }
}
```

```
<img src="<%= site.root %>/images/logo.png" alt="{pm-logo-alt(header)}" />
```

Reply:
It works for img alt text but the same don't work for `aria-label`:
```
// header.json
{
  "pm-hamburger-aria-label": {
    "description": "hamburger aria-label text",
    "message": "Open primary navigation"
  }
}

<button id="hamburger" class="cell2 end desktop-hide" aria-label="{pm-hamburger-aria-label(header)}"></button>
```

Reply:
Only "title" and "alt" seem to be whitelisted, supposedly it should be trivial to add `aria-label`. https://github.com/cmints/cmints/blob/master/lib/i18n.js#L14

Reply:
But on the other hand, if you are using translation string inside of the attribute it should work. Please submit your code so I can see what might be the issue.

Reply:
Meeting notes: Narek will push his changes, might be a Windows issue(hope not).

Reply:
I's done. It was typing issue, I will push the result.